6MI0800193/Week1/Task2.cpp

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+using namespace std;
+
+struct Tuple {
+    int first;
+    int second;
+};
+
+void read_tuple(Tuple& x)
+{
+    cin >> x.first >> x.second;
+}
+
+bool compare_tuples(const Tuple& x, const Tuple& y) // x > y -> true
+{
+    return (x.first > y.first) || ((x.first == y.first) && (x.second > y.second));
+}
+
+Tuple get_biggest(const Tuple tuples[100])
+{
+    Tuple biggest = tuples[0];
+    for (size_t i = 1; i < 100; i++)
+    {
+        if (compare_tuples(tuples[i], biggest))
+        {
+            biggest = tuples[i];
+        }
+    }
+    return biggest;
+}
+
+void swap(Tuple& x, Tuple& y)

```

Comment:
You know you have the [std::swap](https://en.cppreference.com/w/cpp/algorithm/swap) too, right ?